src/Modules/UngradedModules/FreeMod.jl

```diff
@@ -308,3 +308,67 @@ end
 
 rels(F::FreeMod) = elem_type(F)[]
 
+function syzygy_generators(
+    g::Vector{T};
+    parent::Union{<:FreeMod, Nothing}=nothing
+  ) where {T<:FreeModElem}
+  isempty(g) && return Vector{T}()
+  F = Oscar.parent(first(g))
```

Comment:
```suggestion
  F = parent(first(g))
```
there are more

Reply:
The `Oscar.` qualifiers are necessary because of the shadowing from the keyword argument.

Reply:
I see, thanks